src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.sln

```diff
@@ -19,6 +19,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{2648E357
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.ApprenticeFeedback.MockApis", "SFA.DAS.ApprenticeFeedback.MockApis\SFA.DAS.ApprenticeFeedback.MockApis.csproj", "{56EEE787-5656-44A8-AD60-F6D1AD491B2D}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.SharedOuterApi.Apprentice.GovUK.Auth", "..\Shared\SFA.DAS.SharedOuterApi.Apprentice.GovUK.Auth\SFA.DAS.SharedOuterApi.Apprentice.GovUK.Auth.csproj", "{3922E528-51C0-4FD2-9BCB-8753F0F60202}"
```

Comment:
This is curious - why should this have been added here - was your branch synced with master when you started?

Reply:
i was getting compilation errors and it implied this project needed to be included?

Reply:
looks like this is not required.  i think it was related to my dev environment